docs/tutorials/multimodal/matching/image_text_matching.md

```diff
@@ -1,12 +1,12 @@
-# Image<-->Text Matching with AutoMM
+# Image<-->Text Semantic Matching with AutoMM
```

Comment:
We can consider to use `Image-Text Matching with AutoMM`

Reply:
Done.